src/Model/CollectionInfo.php

```diff
@@ -63,6 +63,7 @@ public function __debugInfo()
      */
     public function getCappedMax()
     {
+        /* The MongoDB server might return this number as an integer or float */
```

Comment:
Can use `//` for single-line comments -- unless we are using `/*` in this fashion elsewhere.

Reply:
I had used ``/*`` ... ``*/`` for PHPLIB-376 in src/Operation/CountDocuments.php too — couldn't really see other inline comments.